docs/docsite/rst/dev_guide/testing.rst

```diff
@@ -7,14 +7,15 @@ Testing Ansible
 .. contents:: Topics
    :local:
 
-Introduction
-============
 
-This document describes:
+Why test your Ansible contributions?
+====================================
+
+If you're a developer, one of the most valuable things you can do is look at the GitHub issues list and help fix bugs, since we almost always prioritize bug fixing over feature development.
+Even for non-developers, helping to test pull requests for bug fixes and features is still immensely valuable.
+
+Ansible users who understand writing playbooks and roles should be able to test their work.  GitHub pull requests with various types of tests that show bugs in action will help ensure that your contributions will be on track to be reviewed.  Read on to learn how Ansible is tested, how to test your contributions locally, and how to extend those testing capabilities.
```

Comment:
I've read the 2nd sentence in this paragraph a couple of  times and I'm not sure what it's trying to say. Is it saying that GitHub PRs get a batch of automated testing (aka shippable results)? Or that the contributor should test locally to ensure the PR passes shippable so it can be reviewed w/o waiting for the contributor to fix those shippable errors?

Reply:
* PRs that include tests are more likely to be merged
* Contributors MUST test their PRs and show evidence in the PR